playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2

```diff
@@ -290,6 +290,10 @@ error_page {{ k }} {{ v }};
     try_files $uri @proxy_to_lms_app;
   }
 
+  location /v1/accounts/gdpr_retire_users {
```

Comment:
Can you add a comment on top of this explaining that the endpoint uses OAuth for authentication instead?

Reply:
Addressed 👍🏼